tests/integration/test_failure_output.py

```diff
@@ -52,11 +52,10 @@ def test_failure_output_after_state_change_for_asc(self, single_failing_test_for
 
         # Setup
         single_failing_test_for_asc.tests[0].state = 'passed'   # Change the state of the test to 'passed'
-        single_failing_test_for_asc.invoke_zigzag(force_clean_up=False)     # Re-run ZigZag
+        single_failing_test_for_asc.assert_invoke_zigzag(force_clean_up=False)     # Re-run ZigZag
         test_runs = single_failing_test_for_asc.tests[0].qtest_test_runs
 
         # Test
-        single_failing_test_for_asc.assert_queue_job_complete()
```

Comment:
Implemented this for convenience when I want to re-run without cleaning up first. Turns out this pattern is pretty common.